forge-gui/res/cardsfolder/upcoming/possession_engine.txt

```diff
@@ -0,0 +1,8 @@
+Name:Possession Engine
+ManaCost:3 U U
+Types:Artifact Vehicle
+PT:5/5
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChange | TriggerDescription$ When CARDNAME enters, gain control of target permanent.
+SVar:TrigChange:DB$ GainControl | ValidTgts$ Creature.OppCtrl | TgtPrompt$ Choose target creature an opponent controls | AddKWs$ HIDDEN CARDNAME can't attack or block. | Duration$ AsLongAsControl

```

Comment:
Duration$ in GainControl still has different syntax a bit, also the KW only last until EOT